src/sentry/api/urls.py

```diff
@@ -1411,7 +1411,7 @@
                     name="sentry-api-0-organization-releases-stats",
                 ),
                 url(
-                    r"^(?P<organization_slug>[^\/]+)/releases/(?P<version>[^/]+)/$",
+                    r"^(?P<organization_slug>[^\/]+)/releases/(?P<version>[^/]*)/$",
```

Comment:
Accepting no version is gonna cause this route to collide with the OrganizationReleasesEndpoint 

https://github.com/getsentry/sentry/pull/36452/files#diff-0568d2568ed366405103d213b7ba1291a283f1e517375416fb758ffd41f308f0R1404